modules/apps/frontend-js/frontend-js-components-web/src/main/java/com/liferay/frontend/js/components/web/internal/servlet/taglib/FrontendComponentsTopHeadDynamicInclude.java

```diff
@@ -53,10 +56,17 @@ public void include(
 		StringBundler sb = new StringBundler(5);
 
 		sb.append("<script>var Liferay = window.Liferay || {};");
+
 		sb.append("Liferay.__FF__ = Liferay.__FF__ || {};");
-		sb.append("Liferay.__FF__.enableClayTreeView = ");
-		sb.append(_ffFrontendJSComponentsConfiguration.enableClayTreeView());
-		sb.append(";</script>");
+
+		_createRuntimeFeatureFlag(
+			sb, "enableClayTreeView",
+			_ffFrontendJSComponentsConfiguration.enableClayTreeView());
+
+		_createRuntimeFeatureFlag(
+			sb, "enableCustomDialogs", _shouldEnableCustomDialogs());
```

Comment:
`_createRuntimeFeatureFlag` was created to avoid repetitions when trying to implement feature flags. However, it depends on a validator function and when the number of the FF on this class might increase, we will have tons of validator functions there. I would receive feedbacks regarding this as well suggestions for a more scalable way to setup "Frontend Feature Flags"

Reply:
I think this validator number can be decreased to just one function call when we move completly to use GetterUtil patterns instead of using osgi/configs. We may enter just one string there, containing the property name as `feature.flag.something` and then it will validate using GetterUtil...